membership-attribute-service/test/services/PaymentDetailsForSubscriptionTest.scala

```diff
@@ -90,10 +90,10 @@ class PaymentDetailsForSubscriptionTest(implicit ee: ExecutionEnv) extends Speci
       val contact = TestContact(randomId("identityId"))
       val paymentService = mock[PaymentService]
       val paymentDetailsForSubscription = new PaymentDetailsForSubscription(paymentService)
-      val expectedPaymentDetails = PaymentDetails(friend)
+      val expectedPaymentDetails = PaymentDetails(staff)
```

Comment:
I've  switched all the friend tests to staff for now, although we can almost certainly get rid of staff too